fuse/dir.go

```diff
@@ -246,12 +246,16 @@ func (d *Dir) Create(
 
 	// Handler execution. 'Open' handler will create new element if requesting
 	// process has the proper credentials / capabilities.
-	err := handler.Open(ionode, handlerReq)
+	nonSeekable, err := handler.Open(ionode, handlerReq)
 	if err != nil && err != io.EOF {
 		logrus.Debugf("Open() error: %v", err)
 		return nil, nil, err
 	}
+
 	resp.Flags |= fuse.OpenDirectIO
+	if nonSeekable {
```

Comment:
My understanding is that none of the procfs nodes are seekable, so what about setting these flag for all?

Reply:
Turns out some are; without this (i.e., if we mark all as non-seekable) the sysbox containers with systemd do not start because the systemd sysclt service fails to configure some sysctls (not sure which ones though). With this, things work as expected.

Reply:
interesting, didn't know...